crates/sparrow-qfr-tool/src/chrome_tracing/trace_event.rs

```diff
@@ -124,7 +124,7 @@ pub(crate) enum Event {
 #[derive(Serialize, Eq, PartialEq, Debug, Default)]
 #[allow(dead_code)]
 pub(crate) enum EventScope {
-    /// The event will be drawn from the top to bottom of the timeline.
+    /// The event will be drawn from the top to bottom of the timestream.
```

Comment:
This was referring to the "timeline" in the chrome tracing view. Probably don't want to change.